src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -59,11 +60,6 @@ public class Win32PlatformOptions
         /// GPU rendering will not be enabled if this is set to false.
         /// </remarks>
         public bool? AllowEglInitialization { get; set; }
-        
-        public IList<string> EglRendererBlacklist { get; set; } = new List<string>
-        {
-            "Microsoft Basic Render"
-        };
```

Comment:
Why don't we need it anymore?

Reply:
IIRC it was needed as a workaround for device loss caused by RDP. Since we now properly handle device loss it's not required anymore.